docs: add issue templates (#1034)
iawia002/lux: .github/ISSUE_TEMPLATE/can-not-download-video.md

---
name: Can not download video
about: If you find that a website download is not working
title: "[download fail]: the website name"
labels: bug
assignees: ''

---

<!-- 请输入网站的名称 -->
<!-- enter the website name -->

**Website name**: name

<!-- 你的操作系统 -->

> <!-- your operate system -->

Reviewer: ```suggestion
<!-- your operating system -->
```

> **OS:**: window/linux/macOS

Reviewer: ```suggestion
**OS:**: Windows/Linux/macOS
```

<!-- 请输入下载时显示的错误信息 -->

> <!-- enter the error message when download -->

Reviewer: ```suggestion
<!-- enter the error message when downloading -->
```

**Stack overflow**

```
error message here
```

<!-- 如果能提供截图，则对于解决你的问题非常有帮助 -->

> <!-- if applicable, add screenshots to help explain your problem. -->

Reviewer: ```suggestion
<!-- If you can provide screenshots, it will be very helpful to solve your problem. -->
```
